### PR TITLE
Update docs about remote development and debugging

### PR DESCRIPTION
**Pull request by TongboZhang**
Update docs about remote development and debugging

### CODE REVIEW

File: docs/user/remote_develeoment.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+This can only work in PyCharm Professional, users can configure PyCharm `Deployment` and `Project Interpreter` according to the following animation.
+
+![](./imgs/configure_pycharm.gif)
+
+For more details, please refer to [PyCharm Remote Development](https://www.jetbrains.com/help/pycharm/remote-debugging-with-product.html).
+
+## Others
+
+### PAIPDB
```
Comment (abuccts) [inline docs/user/remote_develeoment.md]: change to PDB?

File: docs/user/remote_develeoment.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+This tool contains two debug modes:
+- Start remote debugging when job task starts -
```
Comment (abuccts) [inline docs/user/remote_develeoment.md]: trailing -

File: docs/user/remote_develeoment.md (hunk 1 above)
Comment (TongboZhang) [reply docs/user/remote_develeoment.md]: Changed

File: docs/user/remote_develeoment.md (hunk 2 above)
Comment (TongboZhang) [reply docs/user/remote_develeoment.md]: Changed

**Pull request merged**